conformance/submission_details_template.txt

```diff
@@ -0,0 +1,105 @@
+##############################################################################
+#
+# SYCL Conformance Submission Details Template
+# Source: https://github.com/KhronosGroup/SYCL-CTS
+# Version: 1.0
+#
+##############################################################################
+#
+# This form should be filled in and included in the conformance submission
+# as the file submission_details.txt.
+#
+# Lines beginning with "#" are comments to assist in filling out the template.
+#
+##############################################################################
+
+##############################################################################
+#
+# Submission information that needs to be copied onto the web submission form.
+#
+##############################################################################
+
+# Contact details.
+#
+# Contact details for individual making the conformance submission.
+#
+Company: 
+Contact name:
+Contact email:
+Contact telephone:
+
+# Version of the SYCL specification being tested.
+#
+# Specification version of SYCL being tested, can be either 1.2.1 or SYCL.
+#
+SYCL Version:
+
+# Conformance submission details.
+#
+# Details of the conformance submission, must specify a specific SYCL
+# implementation and the environment in which the tests were run, including the
+# host architecture, operating system and toolchain and the target SYCL
+# backend, platform and device.
+#
+SYCL implementation and version:
+Host architecture:
+Host operating system and version:
+Host toolchain and version:
+Target SYCL backend:
+Target SYCL platform:
+Target SYCL device:
+
+# Conformant products.
+#
+# Details of all conformant products which derive their conformance from the
+# conformance submission, including that of the conformance submission itself.
+# Must specify all supported SYCL implementation versions, host architectures,
+# operating systems and toolchains, and all supported SYCL backend, platform
+# and device combinations.
+#
+SYCL implementation version(s)
```

Comment:
Should this have a colon at the end of it?

Reply:
Yes thanks, fixed in https://github.com/KhronosGroup/SYCL-CTS/pull/803/commits/a230561300bb379a65616c5ab38ec8bea2886fda.